main.py

```diff
@@ -1,12 +1,203 @@
+        "6": checkout_video,
+        "7": checkin_video,
+        "8": list_customers,
+        "9": get_customer_info,
+        "10": list_customers_rentals,
+        "11": create_customer,
+        "12": update_customer,
+        "13": delete_customer,
+        "20": "Quit"
+    }
+
+
+    print() # Linebreak
+    for key, action in options.items():
+        action_desc = action if type(action) is str else action.__doc__
+        print(f'Option {key}. {action_desc}')
+    print()  # Linebreak
+
+    return options
+
+
+def list_videos():
+    """List all videos"""
+
+    print(*video_store.list_videos(), sep='\n')
+
+
+def create_video():
+    """Create a video"""
+
+    print("Great! Let's create a new video.")
+    title = input("What is the title of your video? ")
+    total_inventory = input("How many copies of this movie are we adding? ")
+    release_date = input("What was the movie's release date? ")
+    response = video_store.create_video(title=title, total_inventory=total_inventory, release_date=release_date)
+    print("New video id:", response["id"])
+
+
+def select_video():
+    """Select a video"""
+
+    select_by = input("In order to select a video, write out title or id:  ")
+    if select_by == "title":
+        title = input("Which title would you like to select? ")
+        video_store.get_video(title=title)
+    elif select_by == "id":
+        id = input("Which video id would you like to select? ")
+        if id.isnumeric():
+            id = int(id)
+            video_store.get_video(id=id)
+    else:
+        print("Could not select. Please enter id or title.")
+    if video_store.selected_video != None:
+        print("Selected video: ", video_store.selected_video)
+
+
+def update_video():
+    """Update selected video"""
+
+    print(f"Great! Let's update the video: {video_store.selected_video}")
+    title = input("What is the new title of your video? ")
+    release_date = input("What is the release date of your video? ")
+    total_inventory = input("What is the new total inventory?")
+    response = video_store.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+    print("Updated video:", response)
+
+
+def delete_video():
+    """Delete a selected video"""
+
+    id = input("Enter video id for deletion: ")
+    if id.isnumeric():
+        id = int(id)
+        video_store.delete_video(id=id)
+    print(f"Video id {id} has been deleted.")
+    pprint.pprint(video_store.list_videos())
+
+
+def checkout_video():
+    """Check out a video"""
+
+    print(f"Great! Let's grab some customer information for rental video: {video_store.selected_video}")
+    customer_id = input("What is your id?")
+    video_id = input("What is the video_id you want to check out?")
+    due_date = input("when is the video due?")
+    video_store.check_out_video(customer_id=customer_id, video_id=video_id, due_date=due_date)
+    print(f"Customer {customer_id} checked out {video_id}, {due_date}")
```

Comment:
I'm a little confused, first I had to select a video then I'm asked for the customer and video id separately.  Shouldn't you just use the selected video?